roles/customize_home/defaults/main.yml

```diff
@@ -0,0 +1 @@
+custom_bootstrap_script: 'bootstrap'
```

Comment:
Should be `customize_home_custom_bootstrap_script` to follow namespacing conventions.